src/analyses/goto_rw.cpp

```diff
@@ -224,12 +228,15 @@ void rw_range_sett::get_objects_member(
 
   const struct_typet &struct_type=to_struct_type(type);
 
-  range_spect offset=
-    to_range_spect(
-      member_offset_bits(
-        struct_type,
-        expr.get_component_name(),
-        ns));
+  auto offset_bits =
+    member_offset_bits(struct_type, expr.get_component_name(), ns);
+
+  range_spect offset;
+
+  if(offset_bits.has_value())
+    offset = to_range_spect(*offset_bits);
+  else
+    offset = -1;
```

Comment:
Nit pick: use `? :` as above instead of the `if ... else`

Reply:
Merged with next branch